web3/_utils/module_testing/go_ethereum_admin_module.py

```diff
@@ -46,19 +46,27 @@ def test_admin_peers(self, web3: "Web3") -> None:
         assert len(result) == 1
 
     def test_admin_start_stop_rpc(self, web3: "Web3") -> None:
-        start = web3.geth.admin.start_rpc("localhost", 8545)
-        assert start
-
         stop = web3.geth.admin.stop_rpc()
-        assert stop
+        assert stop is True
 
-    def test_admin_start_stop_ws(self, web3: "Web3") -> None:
-        start = web3.geth.admin.start_ws("localhost", 8546)
-        assert start
+        start = web3.geth.admin.start_rpc('localhost', 8548)
+        assert start is True
 
+        with pytest.warns(DeprecationWarning):
+            stop = web3.geth.admin.stopRPC()
+            assert stop
```

Comment:
assuming the rest of these start and stop functions return `True`, we can keep the pattern going